core/image_linux.go

```diff
@@ -41,7 +41,7 @@ func (ds *dockerService) ImageFsInfo(
 		return nil, err
 	}
 
-	bytes, inodes, err := dirSize(filepath.Join(info.DockerRootDir, "image"))
+	bytes, inodes, err := dirSize(filepath.Join(info.DockerRootDir, info.Driver))
```

Comment:
This assumption is currently accurate, and I don't think we'll be changing it.

Still, it would be good to link to the code that enforces this:
https://github.com/moby/moby/blob/6ce5aa1cd5a46e02ffaa3fe0c794642b35601676/daemon/graphdriver/driver.go#L162-L164

This code also will not correctly account for the (currently experimental) containerd snapshotter storage backend that we are working to ship by default in a future Moby project release. I'm not quite sure what the right thing to do here is yet, as `dirSize` won't work given that we use a containerd namespace but are not necessarily the sole client of the containerd instance.

It might be necessary to enumerate snapshots and call `Usage()` on each of then, which means that the caching work @xinfengliu is doing will be highly relevant here in the future.